aio/tools/examples/run-example-e2e.js

```diff
@@ -194,13 +192,7 @@ function runProtractorSystemJS(prepPromise, appDir, appRunSpawnInfo, outputFile)
 
         // Start protractor.
         console.log(`\n\n=========== Running aio example tests for: ${appDir}`);
-        const spawnInfo = spawnExt(
-            'yarn',
-            [
-              'protractor', PROTRACTOR_CONFIG_FILENAME, `--specs=${specFilename}`,
-              '--params.appDir=' + appDir, '--params.outputFile=' + outputFile
-            ],
-            {cwd: SHARED_PATH});
+        const spawnInfo = spawnExt('yarn', [ 'protractor' ], {cwd: appDir});
```

Comment:
I would keep the `outputFile` param, since `run-examples-e2e` uses it to log stuff. It currently points to the same file as `defaultOutputFile` in `boilerplate/systemjs/protractor.config.js`, but this could easily change in the future without us noticing:

```suggestion
        const spawnInfo = spawnExt('yarn', [ 'protractor', '--params.outputFile=' + outputFile ], {cwd: appDir});
```

Reply:
If we always set the value (and to the same value as the default), then I would recommend removing the default, no?

Reply:
Keeping the default value (or skipping logging when there is no `outputFile` specified) would be useful for when we want to test an example locally. I think it is useful to be able to just `cd` into an example directory and run `yarn e2e` (or `yarn protractor`).